introduction.md

```diff
@@ -1,2 +1,7 @@
 Hello I'm Cristian doing "mastering github" on codeschool :)
 my avorite Code School path: Ruby
+* Ruby
+* Git
+* PHP
+* Resque
+* Redis
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?